### DISCUSSION TITLE
Stun dims the screen

### DISCUSSION THREAD

**Discussion by PlasmaUwU** [FakeFishGames/Barotrauma#12360] [Ideas and suggestions]
Recently played the Imposter in Baro
And I noticed one illogical and unnoticed "_problem_"

Real example:
You are an engineer, your goal is to steal the captain's name tag, you manage to stun / knock the captain unnoticed.

you take his tag and leave unnoticed too.
A little later, the captain is noticed by the Medic and brought to consciousness, the captain notices the loss of the name tag and checks the Medic for its presence and finds nothing.
The captain understands that no one except the medic and the engineer touched him. The medic is clean, so it's an engineer...

I don't think this is fair to a traitor, and it’s not very logical that you, being unconscious see everything.

Below are pictures of how this could be fixed by darkening the screen depending on the level of stun, a bonus, for example, if the stun is greater than 3, you also do not see people's names

In general, this can be a very radical solution, so it should be allowed to be disabled in the server settings.
![image](https://github.com/Regalis11/Barotrauma/assets/101663742/a21d0fb6-0b39-4966-9a37-d472d4a7b89c)
![image](https://github.com/Regalis11/Barotrauma/assets/101663742/b055635f-6df1-4b28-8d85-4d8b93e03b90)
![image](https://github.com/Regalis11/Barotrauma/assets/101663742/63a12771-0c04-4bd1-934b-1e36b4a363df)
![image](https://github.com/Regalis11/Barotrauma/assets/101663742/811bda7c-ba9d-47f6-a4ec-dc326eed4a01)
![image](https://github.com/Regalis11/Barotrauma/assets/101663742/8f1ae87f-472a-49a6-bd6c-831c1f2c6728)

**Comment by Hazmat616** [comment]
I've gotta say this mod has some unexpected combat feedback that is more positive than negative. Plus everything going black when you become unconscious is great.
